processes/releasing.md

```diff
@@ -90,32 +93,37 @@ components that are tested in periodic jobs.
 We announce the release in Kubernetes slack on `#cluster-api-baremetal` channel
 and through the `metal3-dev` group mailing list.
 
-Email template:
+Email template, with title `New CAPM3, IPAM and BMO minor releases are out!`
+(adjust as necessary depending on what is being released):
 
 ```text
 Hey folks,
 
-CAPM3 v1.x.y, IPAM v1.x.y and BMO v0.x.y minor releases are out now!
-Release notes can be found here: [1], [2], [3].
+CAPM3 v1.x.y, IPAM v1.x.y, BMO v0.x.y and Ironic-image v24.x.y minor releases are out now!
```

Comment:
I think its safe to say vx.y.z for ironic release atleast since if I understood correctly the major version i.e. 24 is tied to the year of release. So next year it would be v25.x I am guessing. I see two major releases for ironic in small span of time 23.x and 24.x

Reply:
fixed